packages/react/src/ImageSlider/ImageSlider.test.tsx

```diff
@@ -23,17 +23,17 @@ describe('Image Slider', () => {
   const images: ImageSliderImageProps[] = [
     {
       alt: 'Bridge',
-      aspectRatio: 'x-wide',
+      aspectRatio: '16:9',
```

Comment:
This is the default aspect ratio for the Image component – all of these can be removed.

Reply:
Hm, not quite. I think we have a bug here. Aspect ratio is used twice in the ImageSlider, for the Image and for the thumbnail button. 

We gave Image a default aspect ratio, but we didn't do the same for the thumbnail button. Both Image and the thumbnail button use the utility class, so they're broken by this PR anyway. I think we should pass a prop to Image in ImageSlider, and turn the thumbnail button into a component which has the same props as Image and a default aspect ratio.

Reply:
I've done this in my last commits